Packages/packages-lock.json

```diff
@@ -121,6 +147,15 @@
         "com.unity.modules.imgui": "1.0.0"
       }
     },
+    "com.unity.visualeffectgraph": {
+      "version": "10.3.2",
+      "depth": 0,
+      "source": "registry",
+      "dependencies": {
+        "com.unity.shadergraph": "10.3.2"
+      },
+      "url": "https://packages.unity.com"
+    },
     "com.unity.xr.legacyinputhelpers": {
```

Comment:
This three packages doesn't load and compile in unity 2020.2.5f1.
Check this or the way to run this. 
the current Error:
```log
An error occurred while resolving packages:
  One or more packages could not be added to the local file system:
    com.unity.render-pipelines.core: EBUSY: resource busy or locked, open 'D:\Projects\Hadi\Fortress-defense\Library\PackageCache\.tmp4404BnZdlsDGxuR7\copy\Runtime\Volume\VolumeComponent.cs'
    com.unity.shadergraph: EBUSY: resource busy or locked, open 'D:\Projects\Hadi\Fortress-defense\Library\PackageCache\.tmp4404vlUDU0goO1t9\copy\Editor\Generation\Descriptors\KeywordDescriptor.cs'
    com.unity.visualeffectgraph: EBUSY: resource busy or locked, open 'D:\Projects\Hadi\Fortress-defense\Library\PackageCache\.tmp4404M6eXpdjU96xI\copy\Editor\Controls\VFXLabeledField.cs'

A re-import of the project may be required to fix the issue or a manual modification of D:/Projects/Hadi/Fortress-defense/Packages/manifest.json file.
```